internal/impl/sql/integration_test.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/benthosdev/benthos/v4/public/service"
 
 	_ "github.com/benthosdev/benthos/v4/public/components/pure"
+	_ "github.com/benthosdev/benthos/v4/public/components/sql"
```

Comment:
Guess these tests haven't been run in a long time

Reply:
I guess too... I had some troubles with this when I was adding the sqlite but I had no prior experience with this